src/renderer/dx/DxRenderer.cpp

```diff
@@ -521,12 +649,11 @@ try
             }
         }
 
-        if (_retroTerminalEffects)
+        if (_HasTerminalEffects())
         {
             const HRESULT hr = _SetupTerminalEffects();
             if (FAILED(hr))
             {
-                _retroTerminalEffects = false;
                 LOG_HR_MSG(hr, "Failed to setup terminal effects. Disabling.");

```

Comment:
Disabling where now?

Reply:
I've added a bunch of places in the latest commit